### ISSUE TITLE
BB Hub installed but no option to sign in Konto

### ISSUE THREAD

**Issue by moses666** [cobalt232/blackberrymanager#80]
Hi, once again thx for your great Work which allows me to use BB Apps even I left the BB Priv. unfortunately I can no longer use the hub after a wipe.  The installation via the BB Manager works, but when setting up an account, the window disappears to the left and the screen remains white.  Can you fix that?  I would be very grateful.  My device: Xiaomi MI6 with MIUI 11 EU Rome.  thank you

**Comment by cobalt232** [comment]
Hi. Can you provide more details? Is Hub+ Services installed? Where are you getting up the account? In Hub or Hub+ Services? Are all permissions granted? Are all ip blockers turned off?

**Comment by moses666** [comment]
Hi, nice to hear from you. Let me explain: I did have the Hub (Inbox) on my Xiaomi Mi6, but after installing a Custom Rom, the Hub is able to Install, but when it comes to set up an account, the screen turn white an you can see the windows move tu the up or to the left, so I cannot write something. I installed your Manager from your gifthub page, latest version and from there i installed Hub+Services, the Hub Inbox and the passwordkeeper. Of course i did give all permissions an okay, in your Manager, Hub+Services, passwordkeeper and the Hub Inbox, but the issue is always there. I can't tell you if there is an IP Blocker, i didn't see something like that. If you have another question let me know. thx and greets from Austria, Markus

**Comment by cobalt232** [comment]
Is this still unresolved? Could you fix the issue on your side?

**Comment by moses666** [comment]
yes, unfortunately the error still exists ... it is very strange that it only appears on my mobile phone, it works for all others ... may be due to the custom rom
